accounts/serializers.py

```diff
@@ -181,3 +181,43 @@ class Meta:
             "cover_picture_url",
             "created_at",
         ]
+
+
+class FollowActionSerializer(serializers.Serializer):
+    message = serializers.CharField(read_only=True)
+    is_following = serializers.BooleanField(read_only=True)
+    followers_count = serializers.IntegerField(read_only=True)
+
+
+class ProfileUserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ["id", "username", "email"]
+
+
+class ProfileFollowingSerializer(serializers.ModelSerializer):
```

Comment:
Since we are using the same serializer for the following and follower list, we can use more semantic class name like `ProfileFollowerAndFollowingListSerializer` 👍